### PR TITLE
Adds a CL-exclusive pistol, the ES-4.

### PR DESCRIPTION
**Pull request by NewyearnewmeUwu**
# About the pull request
This PR adds a CL- and therefore WY - exclusive pistol. The ES-4 is a Weyland Corp manufactures less-lethal pistol that's essentially a reskin of the 88 mod 4 that only fires less-lethal projectiles. Sprites by the fantastic @thwompa 

# Explain why it's good for the game
The pistol uses distinct blue muzzle flashes for its firing so that it's easier to distinguish it from normal guns, and it comes straight from the Aliens RPG. It also looks cool and makes it harder for the CL to get a lethal firearm straight off the bat.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: Added the ES-4 electrostatic pistol to the CL's safe.
/:cl:

### CODE REVIEW

**Comment by thwompa** [comment]
lets gooo!!!!

**Comment by MusselMain** [comment]
seems like a weapon that will fit ML better, allowing CL to be armed shipside, but making sense lore wise. As it wouldn't sense for civilians aboard a military vessel to be armed unless deploying or on code blue/red.

**Review by harryob — APPROVED** [review]
code looks fine

**Review by FransFeiffer — APPROVED** [review]
Sprites are fine, LGTM

File: code/modules/projectiles/guns/pistols.dm [hunk 1]
```diff
@@ -573,6 +573,53 @@
...
+/obj/item/weapon/gun/pistol/mod88/training
+	current_mag = /obj/item/ammo_magazine/pistol/mod88/rubber
+
+
+/obj/item/weapon/gun/pistol/mod88/flashlight/handle_starting_attachment()
+	..()
+	var/obj/item/attachable/flashlight/flashlight = new(src)
+	flashlight.Attach(src)
+	update_attachable(flashlight.slot)
+
```
Comment (Drulikar) [inline code/modules/projectiles/guns/pistols.dm:585]:
```suggestion
```

This code is duplicated code in above lines, and yes the code executes both.

**Review by Drulikar — CHANGES_REQUESTED** [review]

**Comment by cm13live** [comment]
This pull request has been mentioned on **CM-SS13**. Please provide feedback here:

https://forum.cm-ss13.com/t/weapon-restrictions-in-sop/2387/16

**Pull request merged**